### PR TITLE
Moved solar system settings to proxy object

### PR DESCRIPTION
**Pull request by ojustino**
Is it OK that tab-completion also shows the inherited methods from traitlets' `HasTrait` along with the solar system traits?

Edits to the docs are upcoming.

### CODE REVIEW

**Comment by ojustino** [comment]
I edited the docs to to reflect the proxy object.

I also enabled solar system lighting after testing it and being able to toggle it without issue -- both true and false worked. Are you two (cc @philrosenfield) still having problems with it?

File: docs/views.rst [hunk 1]
```diff
@@ -73,11 +70,11 @@ The Universe view zooms all the way out to the extent of the observed universe:
...
 
-.. note:: If you're ever lost inside a mode, the ``reset_view`` method 
-          backtracks to the mode's initial state and allows you to start your
-          exploration over again.
+.. note:: If you're ever lost inside a mode, backtrack to its initial state by
+          using the :meth:`~pywwt.BaseWWTWidget.reset_view` method.
 
 Panorama view
 -------------
+**Keep or no?**
```
Comment (astrofrog) [inline docs/views.rst]: Is there any harm in keeping this for now?

File: pywwt/ss_proxy.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+from astropy import units as u
+from traitlets import HasTraits, observe, validate, TraitError
+
+from .traits import Bool, Int
+
+class SolarSystem(HasTraits):
+
+    def __init__(self, BaseWWTWidget):
```
Comment (astrofrog) [inline pywwt/ss_proxy.py]: I would avoid using ``BaseWWTWidget`` here as that is the class name (not the instance) - instead use ``base_wwt_widget``

File: docs/views.rst (hunk 1 above)
Comment (ojustino) [reply docs/views.rst]: I don't think so. My knowledge of it is just incomplete because I'm not sure what the mode-specific controls are.

**Pull request merged**